<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Pranit" failures="1" tests="3" name="Login Test" time="0.078" errors="0" timestamp="2021-09-18T17:20:59 IST">
  <testcase classname="Pranit.TestNG" name="setup" time="0.015"/>
  <testcase classname="Pranit.TestNG" name="login" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [hey] but found [hii]">
      <![CDATA[java.lang.AssertionError: expected [hey] but found [hii]
at Pranit.TestNG.login(TestNG.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="Pranit.TestNG" name="teardown" time="0.0"/>
</testsuite> <!-- Login Test -->
